Fixed urls to point directly to downloads

gregcar opened the pull request:
StandardResponsetoRequestforInformationWindowsAzureSecurityPrivacy.docx download links were pointed to a Google search result for download.

azurecla commented:
Hi __@mahedley__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

cjgronlund commented:
@nitinme 
Please check the content change here, and let me know whether I should accept the PR.

nitinme commented:
@mumian 
Passing this on to Jonathan because he has been looking more into the HDI docs lately.

mumian commented:
The article has been updated long time back.  You can close this request.

cjgronlund commented:
@mahedley 
Thank you for this suggestion. The updated article included these ideas.
--Carolyn

gregcar committed:
Fixed urls to point directly to downloads

StandardResponsetoRequestforInformationWindowsAzureSecurityPrivacy.docx download links were pointed to a Google search result for download.
